src/styles/themes/_default-theme.scss

```diff
@@ -179,8 +179,17 @@
     --font-family: 'Source Sans Pro';
     --font-stack: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
         Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+    --font-stack-full: var(--font-family), var(--font-stack);
     --font-size: 16px;
 
+    --anchor-color: var(--primary-color-70);
+
+    --border-radius-xs: 2px;
```

Comment:
nit: We should align the private SASS variable to this ($corner-radius-blah to $border-radius-blah). May be a separate change.

Reply:
roger that. doing now